SU-SWS/saa_victoria_subtheme
# Adapt 363  reorganizing banner files

# READY FOR REVIEW

# Summary
- Add function to add has-card and no-card classes on banners
- split banner style files into individual files per banner

# Review By (Date)
- today real quick :)

# Urgency
- nice to move forward theming banners today

# Steps to Test

1. check out branch
1. check an image banner with text has saa-banner--has-card and without text has saa-banner--no-card 

# Affected Projects or Products
- only this subtheme

# Associated Issues and/or People
- all banner tasks ADAPT-363, ADAPT-356, ADAPT-367 etc


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
